<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoreUiClass</class>
 <widget class="QDialog" name="MoreUiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>说明</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/QuickInput.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="title_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="title_icon_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background:none;border-image:url(:/icon.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="title_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>-1</pointsize>
          <weight>63</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family:&quot;Microsoft YaHei&quot;;font-size:14px;font-weight:500;margin-left:5px</string>
        </property>
        <property name="text">
         <string>说明</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="title_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="title_close_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>20</horstretch>
          <verstretch>20</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>关闭</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{background-color:rgba(128,128,128,0.1);image:url(:/close.png);font-size:12px}QPushButton:hover{background-color:rgba(255,0,0,0.3)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="content_widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="styleSheet">
         <string notr="true">QToolBox&gt;QWidget,QScrollArea,QScrollBar,QScrollBar::sub-line,QScrollBar::add-line{background-color:rgba(0,0,0,0);border:none}QScrollBar::handle{background-color:rgba(128,128,128,0.3);border:none}</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="toolBoxReadme0">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>778</width>
           <height>453</height>
          </rect>
         </property>
         <attribute name="label">
          <string>使用方法</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_17">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&lt;a href=&quot;http://help_qinput.cyk.moe&quot;&gt;详细说明：http://help_qinput.cyk.moe&lt;/a&gt;</string>
            </property>
            <property name="textFormat">
             <enum>Qt::RichText</enum>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_18">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>【设置流程】

时间计算（宏-等待，弹出文字-时长，快捷连点-间隔）：
　10CPS=100（CPS=1000/时长）
　10秒=10000（秒=时长/1000）

Quickinput的使用流程：
　1：前往设置选项卡，设置触发按键等（参考设置说明页）

　2（连点器）：前往功能选项卡，设置快捷连点（参考功能说明页）

　3（创建或录制宏）：前往宏选项卡（参考宏说明页）

　4（设置触发方式）：前往触发选项卡（参考触发说明页）

　5（运行）：按下开启/停止键启用，按下触发键或快捷连点键


【常见问题】

1：如何循环执行宏：
　不要在宏编辑窗口里面添加循环或结束，应当在触发里面设置次数，0或空可以无限运行

2：带有编辑的宏操作：
　带有编辑的宏操作有 循环，按键状态检测，找色，找图。
　这些操作需要点击编辑后把对应宏放在里面才能被正确执行

3：使用了鼠标位置功能在3D游戏里：
　不要使用鼠标位置来控制鼠标在3D游戏里面的移动，使用鼠标移动来控制，调节左右或上下来拖动视角</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_10">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="toolBoxReadme1">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>778</width>
           <height>453</height>
          </rect>
         </property>
         <attribute name="label">
          <string>宏</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>宏的执行过程是从上往下（按键，光标，等待），从外往里（循环，颜色，图像）
创建宏：在宏选项卡点击添加，点击左侧添加的宏，右侧框可修改名称，修改后按回车保存
录制宏：点击录制后弹出录制工具栏，点击开始或快捷键后进行操作，完成后点击结束或快捷键
编辑宏：点击编辑来编辑宏，宏里面的各种操作说明在下面的宏操作类型中
运行宏：通过触发来运行对应的宏，触发说明在下面的触发选项卡中</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="toolBoxReadme2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>460</width>
           <height>267</height>
          </rect>
         </property>
         <attribute name="label">
          <string>触发</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>通过触发来设置宏的运行方式和条件

启用/禁用：
　点击列表的状态一列来启用或禁用宏

触发按键：
　点击一次方框，按下所需按键

触发模式：
　切换：点击一次按键运行，再点击一次停止
　按下：按住按键运行，松开按键停止
　松开：松开按键运行，按住按键停止

触发次数：
　切换模式：触发的宏将会无限运行
　按下/松开模式：触发的宏直到完成对应的次数时停止，当次数设置为0或空，宏将无限运行

屏蔽触发键：
　在Quickinput设置启用时，当前按键不会被系统识别，只会被Quickinput处理</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="toolBoxReadme3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>335</width>
           <height>228</height>
          </rect>
         </property>
         <attribute name="label">
          <string>功能</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_19">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>勾选对应功能右侧的框来启用

快捷连点（设置一个按键的连点）：
　1：点击按键右侧方框，按下所需按键
　2：选择模式：
　　按下：按下当前按键开始连点
　　切换：点击当前按键开始连点，再点击当前按键暂停
　　间隔：每次点击的间隔（1秒=1000）

窗口控制状态（切换到对应窗口才能触发宏）：
　点击窗口最右的框，切换到需要控制的窗口，按两次回车
　当切换到设置的窗口才能触发宏（需要先启用Quickinput）

显示当前时间：
　点击按键右侧方框，按下所需按键
　当按下当前按键将会在屏幕中显示时间，无论Quickinput是否启用</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_8">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="toolBoxReadme4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>274</width>
           <height>267</height>
          </rect>
         </property>
         <attribute name="label">
          <string>设置</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_21">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>主题：选择主题颜色

开启/停止健：控制Quickinput是否启用

录制快捷键：录制/录制窗口时使用

录制使用鼠标轨迹：录制的移动自动启用模拟移动轨迹

默认启用：运行Quickinput自动启用

显示触发状态提示：启用Quickinput会在屏幕弹出文字

启用音效：启用或禁用Quickinput时播放声音

默认隐藏：运行Quickinput时隐藏窗口

禁用窗口放大：不会受系统界面放大的影响

开机启动：通过任务计划启动</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_9">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/QuickInput.qrc"/>
 </resources>
 <connections/>
</ui>
